### PR TITLE
VTCCA-0 adding an error message when not all files are ready to upload

### PR DESCRIPTION
**Pull request by DanielConnelly**

### CODE REVIEW

File: test/controllers/VoaPropertyLinkingSpec.scala [hunk 1]
```diff
@@ -121,13 +121,14 @@ trait VoaPropertyLinkingSpec
...
-  override protected def beforeEach(): Unit = {
-    StubIndividualAccountConnector.reset()
-    StubGroupAccountConnector.reset()
-    StubIdentityVerification.reset()
-    StubPropertyLinkConnector.reset()
-    StubBusinessRatesValuation.reset()
-    StubSubmissionIdConnector.reset()
-    StubPropertyRepresentationConnector.reset()
-  }
+//  override protected def beforeEach(): Unit = {
```
Comment (darrell-plant) [inline test/controllers/VoaPropertyLinkingSpec.scala]: Needs a tidy up?

Comment (PeterPerhac) [inline test/controllers/VoaPropertyLinkingSpec.scala]: commented out code

Comment (DanielConnelly) [reply test/controllers/VoaPropertyLinkingSpec.scala]: updated.

Comment (DanielConnelly) [reply test/controllers/VoaPropertyLinkingSpec.scala]: updated.

**Comment by darrell-plant** [comment]
LGTM 👍

File: conf/messages [hunk 2]
```diff
@@ -707,6 +707,7 @@ declaration.p.3=Once you submit the form you will not be able to go back and mak
 label.declaration=Declaration
 declaration.iAgree=I agree to the above declaration
 declaration.required=You must agree to the declaration to continue
+declaration.file.receipt=Awaiting evidence receipt, please try again in a moment.
```
Comment (PeterPerhac) [inline conf/messages]: I don't think this makes it any clearer for the user... I wouldn't know what that means. I tick a box to declare something, press submit, and back to the same page with "Awaiting evidence receipt ..." I'd be well confused. Has the text come from Karen?

Comment (darrell-plant) [reply conf/messages]: It is clearer. But could be worded better.

**Pull request merged**